app/presenters/tree_builder_action.rb

```diff
@@ -7,7 +7,7 @@ def tree_init_options(_tree_name)
 
   # level 0 - root
   def root_options
-    [t = N_("All Actions"), t]
+    [t = _("All Actions"), t]
```

Comment:
OK, then I'll be not afraid to create merge conflicts in the meantime :wink:. I'm planning to rename all the `icon` keys to `image` in hash nodes, because @epwinchell wants to use the `icon` for fonticons.